spec/support/helpers/form_helpers.rb

```diff
@@ -0,0 +1,31 @@
+module FormHelpers
+  def select_option(css_selector, value)
+    find(:css, css_selector).find(:option, value).select_option
+  end
+
+  def fill_in_player_data(index, player_name:, deck_name:, result:)
+    select_option("#game_game_records_attributes_#{index}_player_id", player_name)
+    select_option("#game_game_records_attributes_#{index}_deck_id", deck_name)
+    select_option("#game_game_records_attributes_#{index}_result", result)
+  end
+
+  def game_state(state)
```

Comment:
Same thing here this method's name is not self-documenting. Aka I don't understand what this method does by just reading the name. Something like `select_game_state` would be better.